### ISSUE TITLE
Warnings not being shown to users

### ISSUE THREAD

**Issue by IanVS** [filestack/filestack-js#335]
I've set a max filesize on my picker, and am finding recently that when choosing a file that exceeds that limit, nothing happens.  It is not accepted (correct behavior), but there's no error shown to users to let them know what happened.  

I've also tried using [`onFileSelected`](https://filestack.github.io/filestack-js/interfaces/pickeroptions.html#onfileselected) to manually show a warning, and that doesn't work either:

```js
filepickerClient
    .picker({
        onFileSelected: (file) => {
            console.log(file.size);
            throw new Error('Not accepted');
        },
    })
    .open();
```

And I see the filesize in the console, but still I don't get any kind of message in the UI.

**Comment by akalongman** [comment]
@IanVS  scroll the browser to top and try again. Looks like its a bug in the positioning of message

**Comment by IanVS** [comment]
@akalongman You're right, the message seems to be positioned at the top of the `body` or something, so it doesn't show up if the page is scrolled down before the picker modal is opened.  The message should be positioned to the top of the viewport (`position: fixed`, I guess).

**Comment by IanVS** [comment]
I don't think this is an `enhancement`, it's a bug if users cannot see error messages and are left to wonder why their files are not uploading, right?

**Comment by akalongman** [comment]
Definitely it a bug.

**Comment by maryfs** [comment]
Hi,
we'll fix it in future release.

**Comment by MuhammadGoharKhan** [comment]
Use this css it will be fixed 

.fsp-notifications__container {
    position: fixed;
}

![image](https://user-images.githubusercontent.com/15608977/87390489-63fbe000-c5c2-11ea-80bf-d636fcc9ded1.png)

**Comment by mrshahrukh123** [comment]
@MuhammadGoharKhan thanks the following worked for me
added !important.
.fsp-notifications__container {
	position: fixed !important;
}

**Comment by edouardmenayde** [comment]
Any news on this ?
